### DISCUSSION TITLE
[Vote ended on 2022-12-16] Removal from Ansible 9: cyberark.pas

### DISCUSSION THREAD

**Discussion by gotmax23** [ansible-community/community-topics#172] [Votes done]
### Summary

I'm starting this vote in accordance with [Removal from Ansible -- Collections not satisfying the Collection Requirements](https://github.com/ansible-collections/overview/blob/main/removal_from_ansible.rst#id41).

On 27 October, I opened an issue (https://github.com/cyberark/ansible-security-automation-collection/issues/46) against cyberark.pas's repository reporting their violation of the release tagging requirements. There has been no acknowledgment of the issue, and it hasn't been resolved, so it's subject to removal from Ansible 9. The corresponding community-topics issue (https://github.com/ansible-community/community-topics/issues/168) has been for 8 days.

Note that users can still install the collection with `ansible-galaxy collection install cyberark.pas` and the collection is eligible for re-inclusion if it fixes these issues. 

### Vote

Vote `+1` if you think this collection should be removed from Ansible 9 and `-1` if you think it should stay.

No discussion should happen in this ticket; it's for voting only. Please direct all discussion to https://github.com/ansible-community/community-topics/issues/168. Thanks!

**Comment by gotmax23** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by oraNod** [comment]
+1

**Comment by cybette** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by ssbarnea** [comment]
+1

**Comment by szh** [comment]
-1
